### ISSUE TITLE
Publication to Multiple Sources

### ISSUE THREAD

**Issue by cc-a** [ImperialCollegeLondon/hpc_portal#55]
As discussed in #54 the current behaviour when publishing is as follows:
- Publications are attempted to all currently linked repositories
- Only jobs with no existing publication records can be published

In short, this means if a job is published when only repository A is linked then if repository B is later linked it is then not possible to publish to repository B.

The suggested change is that publication of a job should be possible so long as there is a linked repository against which no publication has yet been made. The interface for this will be provided by including the `Publish` link along with the DOI's of any existing publications in the Repository column of the job list table.

Was this what you had in mind @hrzepa?

**Comment by hrzepa** [comment]
> On 24 Aug 2021, at 22:17, Christopher Cave-Ayland ***@***.***> wrote:
> 
> 
> As discussed in #54 the current behaviour when publishing is as follows:
> 
> 	• Publications are attempted to all currently linked repositories
> 	• Only jobs with no existing publication records can be published
> In short, this means if a job is published when only repository A is linked then if repository B is later linked it is then not possible to publish to repository B.
> 
> The suggested change is that publication of a job should be possible so long as there is a linked repository against which no publication has yet been made. The interface for this will be provided by including the Publish link along with the DOI's of any existing publications in the Repository column of the job list table.
> 
> Was this what you had in mind @hrzepa?
> 

Yes.   At the moment, only two publication repositories are envisioned.

But, if for example a need was identified to add a third, how easy would the interface you have developed be to do this? We have two possibilities

a) A hidden interface, to be added by a developer
b) an exposed interface which could be added by a user.


I suspect a) is by far the easiest?


Henry
